### DISCUSSION TITLE
Nx:run-commands and serverless invoke not working

### DISCUSSION THREAD

**Discussion by johnsonps08** [nrwl/nx#18554] [General]
Tried to upgrade an existing project to Nx 16 and having an issue with Serverless framework with a command that previously worked as expected across multiple projects. See below.

Since the upgrade, all the commands using `npx serverless invoke` no longer complete successfully. Console logs are visible as expected but it gets to the end of the Lambda handler and nothing happens. The CI/CI pipeline we're using eventually timesout but when trying to debug this locally the terminal seems to hang for as long as you're willing to leave it open for (overnight in at least my case). Has anyone else bumped into similar behavior? Does anyone know if something would be different between Nx 14 and 16 that would cause serverless invoke behavior to be different?

```
   "invoke": {
      "executor": "nx:run-commands",
      "options": {
        "commands": [
          {
            "command": "npx serverless invoke -f migrate --stage {args.stage} --aws-profile asyouare_{args.stage}"
          }
        ],
        "cwd": "apps/asd-virtual-api",
        "parallel": false
      }
    }

```

**Comment by AdamH4** [comment]
I faced same issue, but turns out I had to add data to function a suddenly function was called. So my problem was not with Nx but with serverless. Check if serverless version was not updated

```bash
sls invoke [local] --data '{}'
```
